app/Services/Camt/TransactionFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\Camt;
+
+use Genkgo\Camt\DTO\Message;
+use Genkgo\Camt\Camt052\DTO\Statement as Statement052;
+use Genkgo\Camt\Camt053\DTO\Statement as Statement053;
+use Genkgo\Camt\DTO\Entry;
+
+
+class TransactionFactory
+{
+    public static function create(string $camtType, Message $msg, object $levelB, Entry $entry, array $splits): AbstractTransaction
+    {
+        // Read Config heres        
+
+        if ('052' == $camtType) {
+            return new TransactionCamt052($msg, $levelB, $entry, $splits);
+        }
+
+        if ('053' == $camtType) {
+            return new TransactionCamt053($msg, $levelB, $entry, $splits);
+        }
+        // TODO what to do in case of "no valid camtType"?
```

Comment:
Throw big fat error is my best guess, people may complain and then we can debug further.